File: src/Shared/Students.Models/EducationForm.cs
namespace Students.Models;

/// <summary>
/// Форма обучения
/// </summary>
public class EducationForm
{
    /// <summary>
    /// Id формы
    /// </summary>
    public Guid Id { get; set; }
    /// <summary>
    /// Имя формы обучения
    /// </summary>
    public string? Name { get; set; }
}

File: src/Client/src/storage/catalog-configs/education-form.ts
import {
  useGetAllAsync,
  useGetAllPagedAsync,
  useGetOneByIdAsync,
  useAddOneAsync,
  useEditOneAsync,
  useRemoveOneAsync,
} from '../crud/educationFormCrud';
import { educationFormModel } from '../models/index';

export default {
  detailsLink: 'educationForm',
  hasDetailsPage: false,
  serverPaged: false,
  properties: educationFormModel,
  crud: {
    useGetAllAsync,
    useGetAllPagedAsync,
    useGetOneByIdAsync,
    useAddOneAsync,
    useEditOneAsync,
    useRemoveOneAsync,
  },
  columns: [
    {
      title: 'Форма образования',
      dataIndex: 'name',
      key: 'name',
    },
  ],
  dataConverter: (data) => data,
};
